README

```diff
@@ -83,6 +100,17 @@ export USERLAND_ARCHIVES
 
         $ cd (component-dir) ; gmake publish
 
+    If you build several components, and some are required dependencies for the
```

Comment:
Makefile has 
```
(WS_TOP)/components/$(ENCUMBERED)components.mk:        # Makefile # $(WS_TOP)/components
        @echo "Generating component list..."
        @$(BASS_O_MATIC) --workspace=$(WS_TOP) --components=paths \
                | sed -e 's;^$(shell pwd)/;;' \
                | sed -f components.ignore \
                | sed -e 's;^;COMPONENT_DIRS += ;g' >$@
```
So setting ENCUMBERED variable breaks components.mk generation. It's supposed to be set including $(WS_MAKE_RULES)/encumbered.mk .